koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -727,6 +729,11 @@ def _getOutputImageTemplate(self, **kwargs):
                 have_all_fields = False
         if have_all_fields:
             output_template = "%(name)s-%(version)s-%(release)s-%(architecture)s.tar" % kwargs
+
+        extension = self.osbs().get_compression_extension()
+        if extension:
+            output_template = output_template + extension
```

Comment:
Use string formatting here instead as this can easily fail if we got unicode from osbs and try to append it to str.